assets/large_language_models/prompts/generate_a_discharge_letter/spec.yaml

```diff
@@ -0,0 +1,14 @@
+$schema: https://azuremlschemas.azureedge.net/latest/asset.prompt.schema.json
+
+type: prompt
+name: generateadischargeletter
```

Comment:
please use - to separate, for example, generate-a-discharge-letter. Same comment apply for all other assets

Reply:
fixed